### ISSUE TITLE
Emails to authors gets confused on timezones

### ISSUE THREAD

**Issue by hugocorbucci** [agile-alliance-brazil/submissions#150]
Since day and time are separately printed on the email that authors get regarding their proposal, it shows deadlines with a confusing time.
As times in the DB are stored in UTC but times in communications are supposed to be BRT, the effect gets weird. The email needs to be adjusted to inform the correct timezone.

**Comment by gmotta** [comment]
Agreed. But again, it has been there for a long time, the impact is just a few hours off, right?
Or am I missing anything?

**Comment by hugocorbucci** [comment]
Nope. You're right. It's just a few hours. But it's a source of complaints from authors.
